### DISCUSSION TITLE
Does additional_data still work?

### DISCUSSION THREAD

**Discussion by mattmcgrattan** [uriyyo/fastapi-pagination#1143] [Q&A] [answered]
I've tried passing in some simple information (such as a facet count) via the `additional_data` parameter, as per the documentation, but it does not appear in the return.

e.g. 

```python
async def read_all_forms():
    forms = Form.find_all()
    return await paginate(forms, additional_data={"foo": "bar"})
```

Does not return the additional data in the paginated output. Has it been deprecated?

**Comment by uriyyo** [comment]
Hi @mattmcgrattan,

It should work. Which `paginate` function are you using?

**Reply by mattmcgrattan** [reply, marked as answer]
I worked it out. The documentation isn't clear, I don't think, that I needed to define a new custom page class in order for the new data to appear.

I looked again at your tests https://github.com/uriyyo/fastapi-pagination/blob/30bf4702837b3eb7d629467cfa495f3ff4463c37/tests/test_paginator.py#L75 and was able to work it out from that.

Thank you for the response, though.
